lib/views/home_content/map/post_map.dart

```diff
@@ -0,0 +1,45 @@
+import "package:flutter/material.dart";
+import "package:google_maps_flutter/google_maps_flutter.dart";
+import "package:proxima/viewmodels/home_view_model.dart";
+import "package:proxima/viewmodels/map_view_model.dart";
+
+class PostMap extends StatelessWidget {
+  final MapInfo mapInfo;
+
+  static const postMapKey = Key("postMap");
+
+  const PostMap({
+    super.key,
+    required this.mapInfo,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: GoogleMap(
+        key: postMapKey,
+        mapType: MapType.normal,
+        myLocationButtonEnabled: false,
+        myLocationEnabled: false,
+        zoomGesturesEnabled: true,
+        zoomControlsEnabled: true,
+        scrollGesturesEnabled: true,
+        rotateGesturesEnabled: false,
+        tiltGesturesEnabled: false,
+        initialCameraPosition: CameraPosition(
+          target: mapInfo.currentLocation,
+          zoom: 17.5,
```

Comment:
I'm sorry I meant to only highlight the zoom level. 17.5 could be named initialZoomLevel, so that we can easily modify it later from the top of the file.

Reply:
Fixed with 726ef27